packages/react-scripts/scripts/eject.js

```diff
@@ -114,13 +115,28 @@ inquirer
       fs.mkdirSync(path.join(appPath, folder));
     });
 
+    let addtlDeps = new Map();
```

Comment:
maybe rename this to additionalDeps? it's not obvious at the first time I read it....

add tl deps, what is tl? 😄